R/tm_g_ae_oview.R

```diff
@@ -242,14 +242,14 @@ srv_g_ae_oview <- function(id,
     })
 
     observeEvent(input$arm_var, {
-      ANL <- datasets$get_data(dataname, filtered = FALSE) # nolint
```

Comment:
to reviewer:

`filtered = FALSE` was used only to make `"arm_ref"` selectInput to have all variable choices (including filtered). We assume that variable is a factor - then levels are taken from the factor (